### PR TITLE
feat(vpc): vpc regional private networks doc for public beta

### PR DESCRIPTION
**Pull request by RoRoJ**
### Description
VPC / regional Private Networks doc for public beta

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: network/vpc/troubleshooting/vpc-limitations.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- The only product currently available within the regional VPC is Private Networks.
+- Up to 250 resources can be attached to a regional Private Network.
+- A resource can be attached to up to 8 regional Private Networks.
+- The following resource types can be attached to a regional Private Network:
+  - Instances
+  - Elastic Metal servers
+  - Load Balancers
+  - Public Gateways
+  - Managed Databases for PostgreSQL and MySQL
+- It is currently not possible to attach a resource to a regional Private Network from within that resource's own dashboard in the console. You must use the **VPC** dashboard of the console to attach resources to regional Private Networks.
```
Comment (vramiro) [inline network/vpc/troubleshooting/vpc-limitations.mdx:28]: Add in the restrictions that we cannot attach Redis for the moment

File: network/vpc/troubleshooting/vpc-limitations.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+dates:
+  validation: 2023-03-22
+  posted: 2023-03-22
+categories:
+  - network
+---
+
+This page sets out some current limitations of Scaleway's VPC.
+
+- VPC is currently only available in the Paris region.
```
Comment (vramiro) [inline network/vpc/troubleshooting/vpc-limitations.mdx:18]: Add that at the moment there is only one VPC at the region (The "Default VPC") per project.
It is not possible to create, update or delete it

File: faq/vpc.mdx [hunk 3]
```diff
@@ -9,7 +9,28 @@ hero: assets/private-networks.webp
...
   - [Public Gateways FAQ](/faq/public-gateways/)
   - [Private Networks FAQ](/faq/private-networks/)
-</Message>
\ No newline at end of file
+</Message>
+
+## VPC
+
+### What is the difference between VPC and a regional Private Network ?
+One default VPC (**V**irtual **P**rivate **C**loud) for every available region is automatically created in each Scaleway [Project](/console/my-project/concepts/#project). A VPC offers layer 3 isolation. Currently, at Public Beta stage, VPC is only available for the Paris region.
```
Comment (vramiro) [inline faq/vpc.mdx:20]: A VPC offers layer 3 network isolation

File: faq/vpc.mdx [hunk 4]
```diff
@@ -9,7 +9,28 @@ hero: assets/private-networks.webp
...
+
+A regional Private Network sits inside a VPC, and lets you connect resources across multiple AZs within the VPC's region, to a Private Network. This network offers layer 2 isolation. You can create multiple regional Private Networks within a VPC.
```
Comment (vramiro) [inline faq/vpc.mdx:22]: not sure if I'm misreading, but this sentence gives the impression that a regional private network is connecting somehow other private networks, which is is not the case. 

Instead we can just say that a regional private network is a layer 2 network (as private networks), but they are spawned across all the AZs of the region (while private networks are scoped to the AZ)

File: faq/vpc.mdx [hunk 5]
```diff
@@ -9,7 +9,28 @@ hero: assets/private-networks.webp
...
+
+For now, VPC does not offer automation features, interconnection between different regional Private Networks or multiple layers per region. However, more features are planned in the future to enhance the VPC product.
```
Comment (vramiro) [inline faq/vpc.mdx:24]: I would rephrase something like:

On the future VPC will allow you to define specific routing rules among your private networks, interconnect your VPC with other networks, define network access control lists and more.

File: faq/vpc.mdx [hunk 6]
```diff
@@ -9,7 +9,28 @@ hero: assets/private-networks.webp
...
+
+### Can two different regional Private Networks communicate with each other?
+
+No. A regional Private Network spans multiple AZs of the same region, but does not allow communication between two different regional Private Networks. In the future however, we plan to release a feature to enable this.
```
Comment (vramiro) [inline faq/vpc.mdx:28]: true, tHey cannot communicate out of the box and we would need to have the routing tables.
This is also true for private networks as they are today....

But you can define your own L3 router. This can be done using a public gateway or an instance connected to both private networks. Some people does it. I'm not sure if this is documented somewhere.

File: network/vpc/api-cli/vpc-private-beta.mdx [hunk 7]
```diff
@@ -1,30 +1,29 @@
...
-  validation: 2023-03-07
-  posted: 2021-02-14
+  validation: 2023-03-22
+  posted: 2023-02-14
 categories:
   - network
 ---
 
-VPC Regional Private Networks allow you to create Private Networks that are scoped to a region (Paris, Amsterdam or Warsaw), rather than a specific [Availability Zone](/compute/instances/concepts/#availability-zone) (AZ). This means you can attach resources from multiple AZs of that region, to your Private Network.
+Regional Private Networks allow you to create Private Networks that are scoped to a region (Paris, Amsterdam or Warsaw), rather than a specific [Availability Zone](/compute/instances/concepts/#availability-zone) (AZ). This means you can attach resources from multiple AZs of that region, to your Private Network.
```
Comment (RoRoJ) [inline network/vpc/api-cli/vpc-private-beta.mdx:15]:
```suggestion
Regional Private Networks allow you to create Private Networks that are scoped to a region rather than a specific [Availability Zone](/compute/instances/concepts/#availability-zone) (AZ). This means you can attach resources from multiple AZs of that region, to your Private Network.
```

**Review by vramiro — APPROVED** [review]
Thanks!
